Makefile

```diff
@@ -196,7 +216,9 @@ define WRITE_TOOLCHAIN_CONFIG
 endef
 
 .PHONY: $(BUILD_DIR)/$(NATIVE_SYSTEM)/toolchain.config
-$(BUILD_DIR)/$(NATIVE_SYSTEM)/toolchain.config:
+.PHONY: $(BUILD_DIR)/$(NATIVE_SYSTEM)/unittests/toolchain.config
```

Comment:
The cause of this (and related) modifications is that the parallel native builds can interfere with the unittests build.